### PR TITLE
remove libvlc from codebase entirely

### PR DESCRIPTION
**Pull request by dkanada**
This gets rid of the issues with different architectures and decreases build time, with no loss in functionality. It is also progress towards the plugin, which will be much easier once all excess code is gone.

### CODE REVIEW

**Comment by dkanada** [comment]
I would like to add something like this back in eventually but maybe with something a bit nicer like ExoPlayer, and only if the React Native client is slow enough to warrant more effort over here.

File: platforms/android/AndroidManifest.xml [hunk 1]
```diff
@@ -33,14 +29,6 @@
             <meta-data android:name="android.accounts.AccountAuthenticator" android:resource="@xml/authenticator" />
         </service>
         <service android:name="com.mb.android.media.RemotePlayerService" />
-        <service android:name="com.mb.android.media.legacy.KitKatMediaService" />
-        <service android:name="com.mb.android.media.PlaybackService" />
-        <service android:name="com.mb.android.media.MediaService" />
-        <receiver android:name="com.mb.android.media.legacy.RemoteControlClientReceiver">
-            <intent-filter>
-                <action android:name="android.intent.action.MEDIA_BUTTON" />
```
Comment (JustAMan) [inline platforms/android/AndroidManifest.xml]: care to explain why it's removed?

File: platforms/android/build.gradle [hunk 2]
```diff
@@ -193,26 +193,6 @@ android {
     buildToolsVersion "23.0.2"
 
     if (Boolean.valueOf(cdvBuildMultipleApks)) {
```
Comment (JustAMan) [inline platforms/android/build.gradle]: why keep this `if` when its `then` clause is now empty?

Comment (dkanada) [reply platforms/android/build.gradle]: This file shouldn't even be in source control at all, but we can't remove it until we move all the custom changes to another folder. The end goal is to delete `platforms` entirely.

File: platforms/android/AndroidManifest.xml (hunk 1 above)
Comment (dkanada) [reply platforms/android/AndroidManifest.xml]: I removed all files related to VLC playback, and this was one of them.

Comment (JustAMan) [reply platforms/android/AndroidManifest.xml]: So now Jellyfin won't react to play/pause button, or am I reading this wrong?

Comment (dkanada) [reply platforms/android/AndroidManifest.xml]: No it still works fine, this was a separate media player that is currently unused.

**Comment by dkanada** [comment]
@joshuaboniface it works on my machine, let me know if it doesn't work on yours :) there shouldn't be any issues.

**Pull request merged**